### DISCUSSION TITLE
How to restore quick fixes when using Tabby?

### DISCUSSION THREAD

**Discussion by martinellison** [TabbyML/tabby#3605] [Q&A]
When I'm not using Tabby, if I have (say) an unimported name, (which is a syntax error), I can select 'quick fix', and my IDE will give me some options such as create a `use` line or to qualify the name. But with Tabby running, these quick fixes are no longer available. How can I get them back?

This is in the context of Rust and Codium, but it should apply in any language and IDE.

For example, 

- I have a line of code `let new_var = some_func();` 
- but I haven't coded `use some_crate::some_func;`, 
- so the compiler is giving me an error like "cannot find some_func in this scope", 
- and if I click in the right place, I get a list of "quick fixes", 
- and if I'm not using Tabby one of the quick fixes is  "declare `some_crate::some_func;`" or the like; 
- but Tabby doesn't give me that option.

So, is there an advanced setting that I could specify to get the old behaviour back?

**Comment by wsxiaoys** [comment]
Can you provide screenshots for your IDE with / without tabby on quickfixes?

**Reply by martinellison** [reply]
Sorry, quick fixes with Tabby seems to be working correctly today. So perhaps it was not a Tabby problem after all.

**Reply by wsxiaoys** [reply]
No worries!
